static/css/s2.css

```diff
@@ -12983,6 +12983,151 @@ span.ref-link-color-3 {color: blue}
     width: 67%;
   }
 }
+
+.product {
+  max-width: 600px;
+}
+
+#productsPageContent {
+  margin-top: 225px;
+  margin-inline-start: 68px;
+}
+
+.product hr {
+  border: 1px solid var(--light-grey);
+}
+
+.productsInner {
+  display: flex;
+  align-items: right;
+}
+
+.productsInner img {
+  max-width: 100%; 
+  max-height: 116px;
+  height: auto;
+  display: block;
+  margin-inline-end: 3%;
+}
+
+.productsInner .productsDesc {
+  font: 'Roboto', sans-serif; 
+  color: #666666;
+  font-size: 16px;
+}
+
+.productsDesc p {
+  margin-top: 0;
+}
+
+.productsHeader {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10%;
+}
+
+.productsTitleAndLabel {
+  flex: auto;
+}
+
+.productsHeader .productsTitle {
+  font: 'Roboto', sans-serif; 
```

Comment:
use predefined font --vars not explicit

Reply:
I didn't realize we had these - such a huge help! Fixed here, and will use going forward.